script_editor/app/views/plays/show.html.erb

```diff
@@ -319,159 +339,164 @@
-            <% scenes.each do |scene| %>
-                <% if Nokogiri::XML(scene.to_s).css('head').first != nil %>
-                    <div class="sceneDiv" id="Divs">
-                        <button class="scenehead" id="current-scene" data-cut="false" data-display="true">
-                            <%= Nokogiri::XML(scene.to_s).css('head').first.inner_text %>
-                        </button>
-                        <br>
-                <% end %>
+            <!-- Consider this comment the actDiv start (cause of the if above) -->
                 <script>
-                    document.getElementById('current-scene').id = '<%= escape_javascript "s" + currIndex.to_s %>';
-                    document.getElementById('Divs').id = '<%= escape_javascript "Divs" + currIndex.to_s%>';
+                    document.getElementById('current-act').id = '<%= escape_javascript "a" + currAct.to_s %>';
+                    document.getElementById('Diva').id = '<%= escape_javascript "Diva" + currAct.to_s%>';
                 </script>
-                <% currScene = currScene + 1 %>
-                <% currIndex = currIndex + 1 %>
-
-                <% lines = Nokogiri::XML(scene.to_s).css('//sp') %>
-                <% stages = Nokogiri::XML(scene.to_s).css('stage').to_a %>
-
-
-                <% lines.each do |line| %>
-                    <% #DISPLAY STAGE DIRECTIONS %>
-                    <% #Get line id %>
-                    <% lineN = line.attr("xml:id").to_s.gsub("sp-","").to_f %>
-
-                    <% stages.each do |stage| %>
-                        <% #Get stage direction id %>
-                        <% stageN = stage.attr('xml:id').to_s.gsub("stg-","").to_f %>
-                        <% #If there is a stage dir that comes before the line %>
-                        <% if (stageN < lineN) and (stageN >= lineN-1.0)%>
-                            <% #display stage direction %>
-                            <br>
-                            <button class="stage" data-cut="false" data-display="true">
-                                <%= stage.inner_text %>
+                <% currAct = currAct + 1 %>
+                <% scenes = Nokogiri::XML(act.to_s).css('//div2') %>
+                <% currScene = 1 %>
+                <% scenes.each do |scene| %>
+                    <% if Nokogiri::XML(scene.to_s).css('head').first != nil %>
+                        <div class="sceneDiv" id="Divs">
+                            <button class="scenehead" id="current-scene" data-cut="false" data-display="true">
+                                <%= Nokogiri::XML(scene.to_s).css('head').first.inner_text %>
                             </button>
                             <br>
-                            <% stages.delete(stage) %>
-                        <% end %>
                     <% end %>
+                    <!-- Consider this comment the actDiv start (cause of the if above) -->
+                        <script>
```

Comment:
This is a repeated comment to the one above on line 348, do you mean the start of the scene?